code/ATMOSPHERICS/components/unary/vent.dm

```diff
@@ -1,8 +1,8 @@
 /obj/machinery/atmospherics/unary/vent
 	icon = 'icons/obj/atmospherics/pipe_vent.dmi'
-	icon_state = "base"
-	name = "Vent"
-	desc = "A large air vent"
+	icon_state = "base_pas"
+	name = "Passive Vent"
```

Comment:
```suggestion
	name = "passive vent"
```

Reply:
Atmos things are Proper Nouns for some reason, just following the naming scheme.

Reply:
it's dum

Reply:
change them all in a seperate PR, don't 🗞️ me for it

Reply:
I mean, you were changing the name unatomically, I figured you were cool with this. Whatever.